homeassistant/components/plugwise/binary_sensor.py

```diff
@@ -20,11 +24,19 @@
 )
 from .entity import PlugwiseEntity
 
-BINARY_SENSOR_MAP = {
-    "dhw_state": ["Domestic Hot Water State", None],
-    "slave_boiler_state": ["Secondary Heater Device State", None],
-}
 SEVERITIES = ["other", "info", "warning", "error"]
+BINARY_SENSORS: tuple[BinarySensorEntityDescription, ...] = (
+    BinarySensorEntityDescription(
+        key="dhw_state",
+        name="DHW State",
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
+    BinarySensorEntityDescription(
+        key="slave_boiler_state",
+        name="Slave Boiler State",
```

Comment:
Consider not using the word Slave in the name.

Reply:
Given for the name giving that can be altered, as for the key that is a 1:1 from the device XML (see https://github.com/plugwise/python-plugwise/blob/5a70306b81475c43c1a2227b95d6bc4482bdc03b/userdata/anna_heatpump/core.modules.xml#L312) - we could wrap/change that in the python module.

Reply:
For the key, it  is what it is, but the name, we could just change to:

How about "Secondary Boiler State"?